### PR TITLE
Submissions routes add type assignment

### PR DESCRIPTION
**Pull request by itshui3**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Added submissions typing in submissions route
- [ ] Fixed error occurring when transcription attempts to consume image

### CODE REVIEW

File: src/routes/battles/battles.routes.ts [hunk 1]
```diff
@@ -1,81 +1,128 @@
...
+        //     thisMatch.team = team1;
+        // } else if (match.team2_child1_id === id || match.team2_child2_id == id) {
+        //     const team2 = await getCustomRepository(
+        //         MatchInfoRepository,
+        //         connection()
+        //     ).findMatchInfo(match.team2_child1_id, match.team2_child2_id, cohort.week);
+        //     thisMatch.team = team2;
+        // } else {
+        //     console.log('2nd match check fail');
+        // }
```
Comment (grifmang) [inline src/routes/battles/battles.routes.ts:67]: Can we remove the commented code?

File: src/routes/battles/battles.routes.ts [hunk 2]
```diff
@@ -1,81 +1,128 @@
...
         //destructuring the request
 
-        const { id } = req.user as Child
+        const { id } = req.user as Child;
 
         //const result1 = await getRepository(Submission, connection())
-            //.update({ allocation_point: current_point + storypoint })
-            //.where({ id: story1id })
+        //.update({ allocation_point: current_point + storypoint })
+        //.where({ id: story1id })
```
Comment (grifmang) [inline src/routes/battles/battles.routes.ts:91]: This looks like it can be removed too?

**Review by DeusExMashima — APPROVED** [review]
Everything looks good!

File: src/routes/battles/battles.routes.ts (hunk 1 above)
Comment (DeusExMashima) [reply src/routes/battles/battles.routes.ts:67]: Roger that. Will update in the next push!

File: src/routes/battles/battles.routes.ts (hunk 2 above)
Comment (DeusExMashima) [reply src/routes/battles/battles.routes.ts:91]: Noted!

**Pull request merged**